bin/cmd.js

```diff
@@ -21,9 +21,9 @@ if (argv.h || argv.help) {
 var port = argv.port || argv.p
   , host = argv.host
 
-var server = hr(
-    { permament: argv.permament || argv['1']
-    }
-  )
+var server = hr({
+	permament: argv.permament || argv['1']
+	silent: argv.silent || argv.s
```

Comment:
You are missing a comma here. The code style is comma first, that's why `{` is on the same line as the first property.

```js
{ permament: argv.permament || argv['1']
, silent: argv.silent || argv.s
}
```